framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -154,7 +154,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
         // remove the old binding
         if (old != null && !old.isDisposed() && ids[property] != null)
         {
-          old.removeBinding(ids[property]);
+          this.removeBinding(ids[property]);
```

Comment:
The $$refreshNeeded solves problem 1. I don't understand why the Command is the master for all these properties. That can't be correct, or am I wrong?